src/ui/components/Library/Navigation/TeamButton.tsx

```diff
@@ -71,6 +66,12 @@ export function TeamButton({
   );
 }
 
+export function LibraryIcon(teamType: string) {
```

Comment:
This should be `export function LibraryIcon({ teamType }: { teamType: string })`

Reply:
Fixed!